README.md

```diff
@@ -1,70 +1,172 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
+# :CRYPTOCURRENCY: Table of Contents
+- [:List:CRYPTOCURRENCY](#-Awesome-a-nameabout-projecta)
+  - [:hammer_and_wrench: Built With ](#-built-with-a-namebuilt-witha)
+    - [Tech Stack ](#tech-stack-a-nametech-stacka)
+    - [Key Features ](#key-features-a-namekey-featuresa)
+  - [:computer: Getting Started ](#-getting-started-a-namegetting-starteda)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [:busts_in_silhouette: Authors ](#-authors-a-nameauthorsa)
+  - [:telescope: Future Features ](#-future-features-a-namefuture-featuresa)
+  - [:star:️ Show your support ](#️-show-your-support-a-namesupporta)
+  - [:handshake: Contributing ](#-contributing-a-namecontributinga)
+  - [:pray: Acknowledgments ](#-acknowledgments-a-nameacknowledgementsa)
+  - [:memo: License ](#-license-a-namelicensea)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+   # :book: CRYPTOCURRENCY <a name="about-project"></a>
+  - This project lay the foundations for a Cryptocurrency website.It is a sophisticated platform for users to access comprehensive information regarding the leading 30 cryptocurrencies.
+## :hammer_and_wrench: Built With <a name="built-with"></a>
+### Tech Stack <a name="tech-stack"></a>
+- HTML
+- CSS
+- JAVASCRIPT
+- React
+- Redux
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## Available Scripts
+ ### Key Features <a name="key-features"></a>
+ -  Setting up User-Friendly List Display and Filtering
+ -  Implementing API Integration and Data Display
+ -  Creating Test Cases for Application Reliability
 
-In the project directory, you can run:
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### `npm start`
+<!-- LIVE DEMO -->
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## 🚀 Live Demo <a name="live-demo"></a>
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
 
-### `npm test`
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+- [Live Demo Link](https://cryptocurrencymetrics.onrender.com)
 
-### `npm run build`
+- [presentation video Link](https://www.loom.com/share/52030cd256414d75ab8d2a86311d82b1)
 
```

Comment:
- [ ] As previously suggested, your presentation video should include a clear and objective description of the project. However, in your recorded video, you didn't mention the project's purpose and topic so make sure that you record another presentation that the content of the video includes:

  - a description of the project.
  - a demo of the project features.
  - you should also highlight some interesting piece of code or something you built that you are very proud of.